### DISCUSSION TITLE
Not working in Jupyter notebook?

### DISCUSSION THREAD

**Discussion by drscotthawley** [cifkao/html-midi-player#72] [Q&A]
Hi, when I try the basic suggested code inside a Jupyter notebook, I get just a blank output. Screenshot: 
<img width="1572" alt="Screenshot 2023-11-26 at 5 05 11 PM" src="https://github.com/cifkao/html-midi-player/assets/13925685/a974cdbb-bca3-45a8-8c32-db86b1cb771f">

Can anyone suggest what I might be doing wrong?  I don't see any errors in the Javascript console (shown on the right), just a deprecation warning. 

(Haven't seen Jupyter mentioned in the docs or Discussions, so 🤷‍♂️ .)

**Comment by drscotthawley** [comment]
Got it to at least work in an IFrame: 😄 

<img width="1205" alt="Screenshot 2023-11-26 at 5 35 12 PM" src="https://github.com/cifkao/html-midi-player/assets/13925685/29077283-28e9-43cf-ace0-1e3d488e5984">

...But this seems to only work with midi files stored on the internet; it won't work with local files due to CORS policy. 😢

**Comment by drscotthawley** [comment]
For local files on local servers, you can launch a web server with CORS disabled in the background.  But still not sure how to get this to work on Colab. ...?  Maybe via `gradio`?

**Comment by cifkao** [comment]
I actually made an issue about this already: #14 

I don't know what the solution is, maybe a custom Jupyter widget?

Closing as duplicate of #14 (feel free to put any suggestions there).
